src/translations/fr/translation.json

```diff
@@ -1,300 +1,300 @@
+    "Provisions": "Des provisions",
+    "Armored rigs?": "Blindé rigs?",
+    "Min. slots": "Min. fentes",
+    "3-slot": "3-fentes",
+    "4-slot": "4-fentes",
+    "Recoil": "Recul",
+    "Suppressors": "Suppresseur",
+    "Include Marked": "Inclure marqué",
+    "Group by type": "Regrouper par type",
+    "btc, graphics e.t.c": "btc, graphiques etc.",
+    "Min value": "Valeur minimale",
+    "Tarkov Tools Nightbot integration": "Tarkov Tools Intégration Nightbot",
+    "You can add command to your nightbot to get price check in your twitch / youtube channel chat": "Vous pouvez ajouter une commande à votre nightbot pour obtenir une vérification des prix dans votre chat twitch / youtube",
     "Instructions": "Instructions",
-    "Register at": "Register at",
-    "using your twitch / youtube account": "using your twitch / youtube account",
-    "Go to dashboard": "Go to dashboard",
-    "Click the \"Join Channel\" button": "Click the \"Join Channel\" button",
-    "Make bot - moderator, just type /mod nightbot in your chat": "Make bot - moderator, just type /mod nightbot in your chat",
-    "Go to custom commands": "Go to custom commands",
-    "Press the \"Add command\" button": "Press the \"Add command\" button",
-    "Command: !p or anything you like": "Command: !p or anything you like",
+    "Register at": "Inscrivez-vous sur",
+    "using your twitch / youtube account": "en utilisant votre compte twitch / youtube",
+    "Go to dashboard": "Aller au tableau de bord",
+    "Click the \"Join Channel\" button": "Cliquez sur le bouton \"Rejoindre la chaîne\"",
+    "Make bot - moderator, just type /mod nightbot in your chat": "Make bot - modérateur, tapez simplement /mod nightbot dans votre chat",
+    "Go to custom commands": "Allez aux commandes personnalisées",
+    "Press the \"Add command\" button": "Appuyer sur le bouton \"Ajouter une commande\"",
+    "Command: !p or anything you like": "Commande : !p ou tout ce que vous voulez",
     "Message:": "Message:",
-    "Press \"Submit\"": "Press \"Submit\"",
-    "Big thanks to": "Big thanks to",
-    "for feedback": "for feedback",
-    "Not built": "Not built",
-    "Ammo Chart": "Ammo chart",
-    "Tools": "Tools",
-    "Barter trades": "Barter trades",
-    "Hideout crafts": "Hideout crafts",
-    "Items": "Items",
-    "Traders": "Traders",
+    "Press \"Submit\"": "Appuyez sur \"Soumettre\"",
+    "Big thanks to": "Un grand merci à",
+    "for feedback": "pour les commentaires",
+    "Not built": "Non construit",
+    "Ammo Chart": "Graphique des munitions",
+    "Tools": "Outils",
+    "Barter trades": "Commerces de troc",
+    "Hideout crafts": "Artisanat de cachette",
+    "Items": "Articles",
+    "Traders": "Les commerçants",
     "Prapor": "Prapor",
-    "Therapist": "Therapist",
-    "Fence": "Fence",
-    "Skier": "Skier",
-    "Peacekeeper": "Peacekeeper",
-    "Mechanic": "Mechanic",
-    "Ragman": "Ragman",
+    "Therapist": "Thérapiste",
+    "Fence": "Barrière",
```

Comment:
```suggestion
    "Fence": "Fence",
```

Reply:
I'm not sure Fence is an actually french word, however if it is used in game, people will still be able to understand, as such I am committing this change